test.py

```diff
@@ -1,12 +1,38 @@
-from number.FloatNumber import FloatNumber
+# ckrawczyk
+# mrybarczyk
+
 from number.NumberFabric import NumberFabric
 from number.NumberType import NumberType
+from matrix.MyMatrix import MyMatrix
+import numpy
+import random
 
-print("test")
-
+maxRandomValue = 65536
+minRandomValue = -65536
 fabric = NumberFabric()
 numberType = NumberType.FLOAT
-number = fabric.createNumberFromType(numberType, 5, 1)
-number.print()
-number.add(FloatNumber(10, 2))
-print(number.value)
+
+def createMatrix(size, numberType):
+    a = [[None for i in range (0,size)] for j in range (0,size)]
+    x = [None for i in range (0,size)]
+    for i in range (0,size):
+        nominator = random.uniform(minRandomValue, maxRandomValue - 1)
+        number = fabric.createNumberFromType(numberType, nominator, maxRandomValue)
+        x[i] = number.value
+        for j in range (0,size):
+            nominator = random.uniform(minRandomValue, maxRandomValue-1)
+            number = fabric.createNumberFromType(numberType, nominator, maxRandomValue)
+            a[i][j] = number.value
+    b = numpy.matmul(a,x)
```

Comment:
Zastanawiam sie, czy nie opierdoli nas za użycie czegoś takiego xD Na pewno trzeba dać mu link, czy jakiś odnośnik do tego
Zawsze będzie można zrobić też wlasną metodę, ale na razie to zostawmy, tylko dodaj jakiś spoko koment